.circleci/config.yml

```diff
@@ -223,6 +223,8 @@ jobs:
             -PreleaseKeyPassword=$RELEASE_KEY_PASSWORD
       - store_artifacts:
           path: examples/purchase-tester/build/outputs/apk/latestDependencies/release/purchase-tester-latestDependencies-release.apk
+      - store_artifacts:
+          path: examples/purchase-tester/build/outputs/apk/latestDependencies/debug/purchase-tester-latestDependencies-debug.apk
```

Comment:
Is it possible to merge this with the previous `store_artifacts`?

Reply:
We can store a folder instead of the individual apks, but then we would be storing some extra files that aren't needed and are generated in these folders. Also looks like there is no current support for wildcards here unfortunately, though there is a very old feature request for it: https://discuss.circleci.com/t/wildcards-in-path-definition-for-uploading-artifacts/18992